client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -74,12 +79,31 @@ const StatsCommercialPurchase = ( {
 	from,
 	adminUrl,
 	redirectUri,
+	showClassificationDispute = true,
 }: StatsCommercialPurchaseProps ) => {
 	const translate = useTranslate();
 	const isWPCOMSite = useSelector( ( state ) => siteId && getIsSiteWPCOM( state, siteId ) );
 
 	// The button of @automattic/components has built-in color scheme support for Calypso.
 	const ButtonComponent = isWPCOMSite ? CalypsoButton : Button;
+	const [ isAdsChecked, setAdsChecked ] = useState( false );
+	const [ isSellingChecked, setSellingChecked ] = useState( false );
+	const [ isBusinessChecked, setBusinessChecked ] = useState( false );
+
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+
+	const handleClick = ( event: React.MouseEvent, isOdysseyStats: boolean ) => {
+		// TODO: replace with a support ticket
+		const emailHref = 'mailto:user@example.com?subject=Subject&body=message%20goes%20here';
```

Comment:
I had a second thought, maybe we could link to Jetpack support page ( https://jetpackme.wordpress.com/contact-support/?rel=support ) for now, which is not the best but at least works. We'll see if we could submit a ticket automatically. What do you think?

Reply:
Good idea. I'll change it and limit the width to avoid content jumping.